src/components/cue-point/cue-point.scss

```diff
@@ -0,0 +1,34 @@
+@import '~styles/variables';
+
+.cue-point-container {
+  position: absolute;
+  height: inherit;
+  padding: 12px 0 12px + $progress-bar-height 0;
+  margin: -12px 0;
+
+  &:hover {
+    .cue-point-preview-container {
+      display: block;
+    }
+  }
+}
+
+.cue-point {
```

Comment:
define this class inside `cue-point-container`

Reply:
I don't want to be specific. this makes the default class be stronger than a custom class provided by the app (means the app will need to add `!important`).